src/stories/Library/rich-text/rich-text.scss

```diff
@@ -0,0 +1,84 @@
+// This component is used for WYSIWYG content.
+// E.g. we cannot control classes of the underlying elements, so in this case
+// we will target HTML tags instead.
+.rich-text {
+  @include typography($typo__rich-text-body);
+
+  // Setting a max-width to increase readability for sentences.
+  max-width: $layout__max-width--text;
+
+  a {
+    text-decoration: underline solid black;
+    text-underline-offset: 0.3em;
+    transition: text-underline-offset 0.1s ease-in-out;
+
+    &:hover {
+      color: $color__text-primary-black;
+      text-underline-offset: 0.2em;
+    }
+  }
```

Comment:
Consider whether this should include the styling from `.link-tag`.

If there is a reason we do not do so then I would appreciate a comment mentioning why.

Reply:
Link tag is a different thing - it's a link with an arrow and stuff, and overwriting font-size etc.
This is just a usual inline-link.

But, you're right. This is useful in other cases too.
I'll move this chunk of code into a placeholder of it's own.